### DISCUSSION TITLE
IKEA E1812 Not Supported

### DISCUSSION THREAD

**Discussion by radek-foltyn** [Koenkk/zigbee2mqtt#6258] [Devices]
Hi, can you please ask. Today I tried to connect IKEA E1812 (TRADFRI shortcut button) using zigbee2mqtt (CC2531). And unfortunately it wrote me that the device is not supported. Didn't you meet that, please? 

I tried restarting zigbee2mqq for the whole HA and the problem keeps coming up again.

Thank you for your help

**Comment by m0wlheld** [comment]
[Documentation](https://www.zigbee2mqtt.io/devices/E1812.html) says, it is supported. What version of Z2M are you using?

**Reply by radek-foltyn** [reply]
Current version: 1.17.1-1
but
I also tried Current version: 1.16.1 and it doesn't work here either.

**Reply by scoobydrvr** [reply]
Adding that I'm experiencing the same problem with z2m version 1.27.2 commit: 7dc48fb6. Devices can be added but fail Interview step and show as "not supported".
The documentation page says something about removing it from a binding group but that doesn't seem related to the issue seen here.
